circe/decoupled_pricing/child_appointment_greedy.py

```diff
@@ -394,25 +396,25 @@ def read_file(file_name):
 
 
 def output(msg):
-    """if debug is True, print the msg
+    """if debug is True, logging.debug the msg
     
     Args:
-        msg (str): message to be printed
+        msg (str): message to be logging.debuged
     """
     if debug:
-        print(msg)
+        logging.debug(msg)
 
 def get_resource_data_drupe(MONGO_SVC_PORT):
     """Collect the resource profile from local MongoDB peer
     """
 
     for profiler_ip in profiler_ips:
-        print('Check Resource Profiler IP: '+profiler_ip)
+        logging.debug('Check Resource Profiler IP: %s',profiler_ip)
         client_mongo = MongoClient('mongodb://'+profiler_ip+':'+str(MONGO_SVC_PORT)+'/')
         db = client_mongo.central_resource_profiler
         collection = db.collection_names(include_system_collections=False)
-        logging =db[profiler_ip].find().skip(db[profiler_ip].count()-1)
-        for record in logging:
+        logdb =db[profiler_ip].find().skip(db[profiler_ip].count()-1)
```

Comment:
Practice leaving spaces before and after math operators like =, -, +, etc.  It makes code a lot more readable.

Reply:
sure, thanks!